packages/okta-react/src/withOktaAuth.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+import { useAuth, useAuthState } from './OktaContext';
+
+const withOktaAuth = (ComponentToWrap) => { 
+  // const WrappedComponent = (props) => <ComponentToWrap {...props} auth={auth} authState={authState}/>;
+  return (props) => { 
+    const auth = useAuth();
```

Comment:
`hooks` cannot work in class component, this change may cause the `HOC` fail on class component use case.
https://reactjs.org/docs/hooks-faq.html#should-i-use-hooks-classes-or-a-mix-of-both